src/app/application.cpp

```diff
@@ -407,15 +407,62 @@ void Application::processParams(const QStringList &params)
         return;
     }
 #endif
+    BitTorrent::AddTorrentParams torrentParams;
+    TriStateBool skipTorrentDialog;
 
     foreach (QString param, params) {
         param = param.trimmed();
+
+        // Process strings indicating options specified by the user.
+        
+        if (param.startsWith("@savePath=")) {
```

Comment:
I'll make this change in a little while when I'm free, but I'm curious: what's the advantage of using QLatin1String instead of a string literal? Does it make any difference when you're just dealing with ASCII characters?